go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/conduitio/conduit-connector-protocol v0.4.0
 	github.com/conduitio/conduit-connector-s3 v0.3.0
 	github.com/conduitio/conduit-connector-sdk v0.5.1
-	github.com/conduitio/yaml/v3 v3.2.0
+	github.com/conduitio/yaml/v3 v3.2.1-0.20230306160152-c42fb08ec7ff
```

Comment:
Note to self - merge https://github.com/ConduitIO/yaml/pull/4, release `v3.2.1` and update the dependency before merging this PR.